### PR TITLE
[GOVCMSD7-282] Improve govcms-deploy script

### PR DESCRIPTION
**Pull request by pandaskii**
As discussed, the PR contains the following changes:

- Remove the one-off fix to redirect module legacy issue
- Add a ```drush updb - y``` for PROD db synced across
- Run ```drush updb - y``` before ```drush en stage_file_proxy -y```

### CODE REVIEW

File: .docker/images/govcms7/scripts/govcms-deploy [hunk 1]
```diff
@@ -6,46 +6,39 @@
...
+    
+    # Non production environments.
+    if [[ ! -z "$LAGOON_ENVIRONMENT_TYPE" && "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
+        # Enable stage file proxy post db-import.
+        drush en stage_file_proxy -y
+    fi
+}
+
 # Non production environments.
 if [[ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
```
Comment (AMDS) [inline .docker/images/govcms7/scripts/govcms-deploy]: Do we need to check if the $LAGOON_ENVIRONMENT exist? 
For example, 
if [[ ! -z "$LAGOON_ENVIRONMENT_TYPE" && "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then

File: .docker/images/govcms7/scripts/govcms-deploy [hunk 2]
```diff
@@ -6,46 +6,39 @@
...
-fi
-
-# All valid environments.
-if tables=$(drush sqlq 'show tables;') && [ -n "$tables" ]; then
-  drush updb -y
-fi
+    if tables=$(drush sqlq 'show tables;') && [ -n "$tables" ]; then
+        mkdir -p /app/sites/default/files/private/backups/ && drush sql-dump --ordered-dump --gzip --result-file=/app/sites/default/files/private/backups/pre-deploy-dump.sql
+        common_deploy
+        drush en -y govcms_lagoon && drush dis -y govcms_lagoon; drush pmu -y govcms_lagoon;
```
Comment (AMDS) [inline .docker/images/govcms7/scripts/govcms-deploy]: Why we enable the govcms_lagoon module and then disable and uninstall it?

Comment (simesy) [reply .docker/images/govcms7/scripts/govcms-deploy]: It's just a way to execute operations without it being part of the config of the site.

Comment (simesy) [reply .docker/images/govcms7/scripts/govcms-deploy]: Oh ignore my comment about config - it's drupal 7.

File: .docker/images/govcms7/scripts/govcms-deploy (hunk 1 above)
Comment (simesy) [reply .docker/images/govcms7/scripts/govcms-deploy]: @AMDS for your interest, for Drupal 8 there is a further improved version of this script coming in another PR.
https://github.com/simesy/scaffold-tooling/blob/master/scripts/govcms-deploy

**Pull request merged**